public/about/privacy/index.html

```diff
@@ -300,25 +300,6 @@ <h3>Analytics</h3>
                     </u>
                 </ul>
                 </li>
```

Comment:
The `</ul>` and `</li>` should be swapped here, a very small quirk. 

Because the opening `<li>` tag in **line 294** is closed by the `</li>` on **line 302** which should be before the closing `</ul>` tag